tests-behat/basicexecutor.feature

```diff
@@ -1,3 +1,4 @@
+@basic-executor
```

Comment:
just curious, what is this for?

Reply:
allow to run a specific test instead of the whole suite. usage: bin/behat --tags basic-executor

I use to be able to run specific test before using bin/behat features/basicexecutor.feature but this is not working anymore and could not find a way using the new structure. So I just use tags instead

Reply:
features are now moved to `tests-behat/`, if that work, I think it is better to no duplicate file names

Reply:
try using tests-behat but did not work. Also using tags is quicker...